packages/material-ui-unstyled/src/BadgeUnstyled/BadgeUnstyled.js

```diff
@@ -40,7 +40,6 @@ const BadgeUnstyled = React.forwardRef(function BadgeUnstyled(props, ref) {
       vertical: 'top',
       horizontal: 'right',
     },
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do we need a lint rule to disable eslint-disable for some rules? :grin:

Reply:
Hahahahha, nah it's just me :D